src/main/java/org/calypsonet/terminal/calypso/transaction/SamSecuritySetting.java

```diff
@@ -0,0 +1,19 @@
+/* **************************************************************************************
+ * Copyright (c) 2022 Calypso Networks Association https://calypsonet.org/
+ *
+ * See the NOTICE file(s) distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License 2.0 which is available at http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ************************************************************************************** */
+package org.calypsonet.terminal.calypso.transaction;
+
+/**
+ * Data to manage the security operations of a Calypso SAM transaction.
+ *
+ * @since 1.2.0
+ */
+public interface SamSecuritySetting extends CommonSecuritySetting<SamSecuritySetting> {}
```

Comment:
Here, we will certainely define in the future a method like:

SamSecuritySetting setControlSamResource (CardReader controlSamReader, CalypsoSam controlSam)